superset-frontend/src/setup/setupApp.ts

```diff
@@ -95,6 +95,6 @@ export default function setupApp() {
   window.jQuery = $;
   require('bootstrap');
 
-  // setup appwide custom error messages
+  // set up app wide custom error messages
```

Comment:
notable. `setup` is a noun but this wants the verb-phrase, and there's no particularly good reason to merge `app`+`wide`

Reply:
arguably it should be app-wide.

Reply:
... but I won't lose any sleep over that.